deprecated_apis.txt

```diff
@@ -748,6 +748,7 @@ constructs.Node#uniqueId
 @aws-cdk/aws-ecs.EcsOptimizedAmiProps#windowsVersion
 @aws-cdk/aws-ecs.FargateServiceProps#securityGroup
 @aws-cdk/aws-ecs.SplunkLogDriverProps#token
+@aws-cdk/aws-ecs.ServiceAttributes#fromServiceAtrributes
```

Comment:
Similarly to below, we won't be deprecating anything. No need to have this

Additionally, this file is for functionalities specifically deprecated from `v1` which no longer exist in `v2` - so we wouldn't want to add this anyway 🙂

Reply:
Thank you for your explanations😄